src/server/responseMessage.js

```diff
@@ -0,0 +1,14 @@
+const responseMessage = {
```

Comment:
the keys could be upper case like: `OK_STATUS`, `API_ERROR_MESSAGE` because they are really constant variables and it's a good practice to use upper case variable names for const vars